### DISCUSSION TITLE
Manual vs Scored documentation

### DISCUSSION THREAD

**Discussion by joebowbeer** [aquasecurity/kube-bench#1282] [Questions and Help] [answered]
I created an issue to document the `--unscored` flag: #1281 

In addition, I think some documentation changes are needed regarding Manual vs Scored tests.

There is a lot of confusion about this, and the clearest explanation exists only in the discussions:

* https://github.com/aquasecurity/kube-bench/discussions/906#discussioncomment-872135
* https://github.com/aquasecurity/kube-bench/discussions/772#discussioncomment-868285
* https://github.com/aquasecurity/kube-bench/discussions/595#discussioncomment-2898558

The [references](https://github.com/aquasecurity/kube-bench/blob/main/docs/flags-and-commands.md#output-manipulation-flags) to _Manual_ in the documentation are ambiguous because they can refer either to a `type` meaning the test is not automated, or they can refer to some text in the test description, indicating that the test is not _scored_, even though it may or may not be automated (executable).

There are also a few [tests](https://github.com/aquasecurity/kube-bench/blob/main/cfg/eks-1.1.0/node.yaml#L254) with _(Automated)_ in their description that are not scored.

**Comment by mozillazg** [comment, marked as answer]
@joebowbeer You are right, we should update documents to cover these questions. PR is welcome.
